src/components/Book.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
+    <td>{book.id}</td>
+    <td>{book.title}</td>
+    <td>{book.category}</td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.isRequired,
```

Comment:
- [ ] You have forget the specify the data-type of book. If you checked your console you will get this error:
![Screenshot from 2020-10-08 19-17-25](https://user-images.githubusercontent.com/4246637/95492157-f4b44300-099a-11eb-98a8-019e8920e07b.png)

The same error also in bookList component. To fix this error you can use [`shape`](https://reactjs.org/docs/typechecking-with-proptypes.html).